gettsim/policy_environment.py

```diff
@@ -362,6 +362,20 @@ def _load_parameter_group_from_yaml(date, group, parameters=None):
                     for key in value_keys:
                         tax_data[param][key] = policy_in_place[key]
 
+            # Also load earlier parameter values if this is specified in yaml
+            if "access_different_date" in raw_group_data[param]:
+                if raw_group_data[param]["access_different_date"] == "vorjahr":
+                    date_last_year = date - datetime.timedelta(days=365)
```

Comment:
this is not robust to leap years, isn't it? I'd suggest something like
```python
date_last_year = date(date.year - 1, date.month, date.day)
```

Reply:
good point, will adjust

Reply:
Done!